tests/integration/linux_bridge_test.py

```diff
@@ -537,6 +539,37 @@ def test_disable_vlan_filtering_on_bridge(
             assertlib.assert_state_match(desired_state)
 
 
+def test_bridge_with_unmanaged_ports():
+    bridge_config = _create_bridge_subtree_config([])
+    with linux_bridge(TEST_BRIDGE0, bridge_config):
+        with dummy0_as_slave(TEST_BRIDGE0):
+            current_state = show_only((TEST_BRIDGE0,))
+            bridge_state = current_state[Interface.KEY][0]
+            port_subtree = bridge_state[LinuxBridge.CONFIG_SUBTREE].get(
+                LinuxBridge.PORT_SUBTREE, []
+            )
+
+            assert port_subtree[0].get(LinuxBridge.Port.NAME) == "dummy0"
+
+
+@contextmanager
+def dummy0_as_slave(master):
+    try:
+        exec_cmd(("ip", "link", "add", "dummy0", "type", "dummy"), check=True)
```

Comment:
The `try` block should start *after* this line.

Reply:
Yes, you are right. Done.